GUI/MainInstall.cs

```diff
@@ -96,6 +97,31 @@ public partial class Main
                 installCanceled = true;
             };
 
+            GUI.user.RaiseMessage("About to install...\r\n");
```

Comment:
I haven't made any updates; I just rearranged the pull request to be easier to review. I didn't get to leave my day job yesterday until dark thirty. :(